test/FunctionalTests/ConfigTests/ConfigProvidersWebTests.cs

```diff
@@ -0,0 +1,76 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace EntropyTests.ConfigTests
+{
+    public class ConfigProvidersWebTests
+    {
+        private const string SiteName = "Config.Providers.Web";
+
+        [Theory]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:6300")]
+        public async Task RunSite_AllPlatforms(ServerType server, RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, string applicationBaseUrl)
+        {
+            await RunSite(server, runtimeFlavor, architecture, applicationBaseUrl);
+        }
+
+        [ConditionalTheory]
```

Comment:
Similar opinions on the rest of these with regard to redundant and commented tests.